torch/onnx/symbolic_opset9.py

```diff
@@ -769,18 +769,18 @@ def wrap_with_not(g, input, other):
     return wrap_with_not
 
 
-@wrap_logical_op_with_cast_to('Byte')
```

Comment:
@houseroad

Reply:
Should be ok to remove these `@wrap_logical_op_with_cast_to('Bool')` decorators. These ONNX ops return output with dtype Bool. They were added to be compatible with PyTorch (uint8).

Reply:
I think @BowenBao is right. Also I am wondering whether we should keep wrap_logical_op_with_cast_to function anymore, since it should be used anywhere if I remember correctly.